File: cwd.py
import os
import subprocess
import sys

def print_help():
    print("Usage: cwd [path]")
    print("Copies the current working directory (or specified path) to clipboard")

def copy_to_clipboard(text):
    process = subprocess.Popen('clip', stdin=subprocess.PIPE, shell=True)
    process.communicate(input=text.encode())

def main():
    args = sys.argv[1:]
    
    if any(arg in ('-h', '--help', '/?') for arg in args):
        print_help()
        return 0
    
    path = args[0] if args else os.getcwd()
    folderPath = os.path.abspath(path)
    
    print(folderPath)
    copy_to_clipboard(folderPath)
    return 0

if __name__ == "__main__":
    sys.exit(main())

File: cwd.js
var path = require('path');
var args = process.argv.slice(2);

function print_help() {
    console.log("Usage: cwd [path]");
    console.log("Copies the current working directory (or specified path) to clipboard");
}

if (args.includes('-h') || args.includes('--help')) {
    print_help();
    process.exit(0);
}

var targetPath = args[0] || process.cwd();
var resolvedPath = path.resolve(targetPath);

console.log(resolvedPath);
require('child_process').spawn('clip').stdin.end(resolvedPath);